components/AppBar/src/ColorThemer/MDCAppBarColorThemer.h

```diff
@@ -44,6 +44,22 @@
 + (void)applySurfaceVariantWithColorScheme:(nonnull id<MDCColorScheming>)colorScheme
                     toAppBarViewController:(nonnull MDCAppBarViewController *)appBarViewController;
 
+@end
+
+@interface MDCAppBarColorThemer (ToBeDeprecated)
+
+/**
+ Applies a color scheme's properties to an MDCAppBar.
+
+ @warning This method will soon be deprecated. Consider using @c +applySemanticColorScheme:toAppBar:
```

Comment:
Given our more recent naming decisions, we probably should not encourage folks to use an API that will also be deprecated soon.

The new ones will probably be something like this:
```
+ applyColorScheme:toPrimaryVariantAppBar:
+ applyColorScheme:toSurfaceVariantAppBar:
```

Reply:
These are not new comments - this is only moving existing code to more clearly indicate that it is intended to be deprecated. I'm happy to clarify those comments in a separate change.

Reply:
Ok, then could you please clarify the PR's summary and description (which is currently absent)? From the summary, it seems like you're updating the annotations for deprecated APIs. I would have expected those notes to include the correct new APIs for use.

Reply:
Done.